config.def.h

```diff
@@ -6,6 +6,9 @@ static const unsigned interval = 1000;
 /* text to show if no value can be retrieved */
 static const char unknown_str[] = "n/a";
 
+/* datetime format (see man 3 strftime) */
+static const char datetime_format[] = "%a %d %b %Y %R:%S";
```

Comment:
This should be a macro so that the compiler can perform compile time analysis of the format string which is not possible when the string is stored in a variable

Reply:
hmm, fixable